Tensile/Tests/unit/test_TensileCreateLibrary.py

```diff
@@ -352,9 +353,44 @@ def test_generateMasterFileList():
     assert isinstance(t[1], MasterLibraryMock), "Incorrect type for value."
     assert t[1].data == (idx+2), "Incorrect data."        
 
+
+def test_componentsOfLogicDataAndSolutionsConstruction(initGlobalParametersForTCL):
+    
+    requiredArgs = ["--jobs=2", "/unused/logic/path", "/unused/output/path", "HIP"]
+    
+    with initGlobalParametersForTCL(["--architecture=gfx900"]+requiredArgs):
+        for separateArch in [False, True]:
+            rootPath = Path(__file__).parent/"../test_data"/"unit"/"solutions"
```

Comment:
```suggestion
            rootPath = Path(__file__).parent.parent/"test_data"/"unit"/"solutions"
```
Otherwise, when testing on Windows, it won't have the correct path separators. Alternatively, I believe `.parents[2]` will do the trick.

Reply:
Good catch fixed.